### ISSUE TITLE
Add ical web feed to top of THAT Board

### ISSUE THREAD

**Issue by brettski** [ThatConference/that.us#320]
**Describe the solution you'd like**
Make a button/link/something to allow members to add THAT Board to their calendar using our shared calendar ics feed.

**Comment by asharonbaltazar** [comment]
Would you like the button somewhere here?

![image](https://user-images.githubusercontent.com/58940073/93891623-3661a000-fcb1-11ea-903b-21dcd20de08e.png)

**Comment by brettski** [comment]
> Would you like the button somewhere here?
> 
> ![image](https://user-images.githubusercontent.com/58940073/93891623-3661a000-fcb1-11ea-903b-21dcd20de08e.png)

Hi @asharonbaltazar. Thanks for reaching out. We already have iCal downloads in the details of each future Activity. The though on this is providing a link somewhere to download the iCal feed maintained off all the activities. It is a long, ugly link, so we shortened it to, https://bit.ly/2YRdVxy. I didn't have any idea on where, plus realize, I don't UI much 😛

**Comment by asharonbaltazar** [comment]
Okay, I'll figure out a way

**Comment by asharonbaltazar** [comment]
@csell5 Would like your input as well.
Maybe something like this for desktop? 
![image](https://user-images.githubusercontent.com/58940073/94064888-ed444580-fdaf-11ea-9833-8373d15352e2.png)

Mobile is a bit trickier. Maybe an icon that's a bit more clearer? Any text will clutter that row. 
![image](https://user-images.githubusercontent.com/58940073/94065057-35636800-fdb0-11ea-98db-09526edb27d1.png)

**Comment by csell5** [comment]
Few thoughts here. 

* In the pr that I'm currently finishing up you can see I now have some logic to copy that calendar feed to your clipboard https://thatus-k4tyq7gnr.vercel.app/support/staying-up-to-date. Nothing complicated but, good to know as that's really the right user behavior as you can't just click on the link to subscribe, you have to paste that into your calendar program of choice to subscribe.

* This link is basically useless after a user does it. So rather than it taking up UI space, I thinking we treat it more like an ad. This is something we need to think about designing anyways for other reasons but I'd love to see "ad cards" where we can push messages, ads, whatever and just drop them into the feed. 

* The link alone isn't enough, without some content. The new content card might help with that, but also just copying the link wouldn't do the user any good, which could point to taking them to a new page. Now we could take them to the new help page, but again, once they've done it. useless link.

* as for useless. Maybe we drop something in local storage to hide that if the user has in fact copied it, or selected to never see it again.

I know this complicates things a bit, but we need to surface information such that it doesn't make the rest of the ui become noise that people just don't look at.

**Comment by asharonbaltazar** [comment]
Yeah, you made a lot of good points, especially about "noise." However, doesn't the feed change intermittently whenever there's a new event or something?

**Comment by csell5** [comment]
right now the "feed" is more like a calendar of activities from today forward. in the future we will enhance that to be more feed(ish) as there will be other content types. "Ads" might be the wrong word here as much as it's a different content item that we want to appear in the feed. 

Toasts isn't a bad idea. Maybe for now we drop a message in the Notification Center? That's really what it's for assuming we can start to drive people to look there for updates. Biggest problem with it right now is there isn't any state backing it. We need to build out the backend for it.

As I write this.. Maybe what we really need rather than the ical feed being surfaced is an info card on how to get started (/support/welcome) which in turn already references how to stay up to date.

I'm open to ideas. thoughts?
